app/views/cdo/create/microchip-search.njk

```diff
@@ -34,21 +34,13 @@
     {% endif %}
 
     <form method="POST" novalidate autocomplete="off">
```

Comment:
Are we going to be missing an h1 from this page now?

Reply:
the h1 has been moved to the govukInput.  Both govukInput and govukFieldset allow you to put the page header into those components.  It resolves the requirement always have a label for input fields.